app/errors.py

```diff
@@ -22,6 +22,7 @@ class OTAModules(Enum):
     StandbySlotCreater = 2
     Downloader = 3
     API = 4
+    OTASERVICE = 5
```

Comment:
Enum entry should be UPPER_SNAKE.
https://kamatimaru.hatenablog.com/entry/2021/10/20/230128